File: php/testcode/benchmark_test_00179.php
<?php
require_once __DIR__ . '/shared.php';

function benchmarkTest00179(BenchmarkRequest $req): BenchmarkResponse {
    $count = $req->param('count');
    $output = [];
    exec("ping -c " . intval($count) . " 8.8.8.8", $output);
    return BenchmarkResponse::ok(implode("\n", $output));
}

File: rust/testcode/benchmark_test_00179.rs
pub fn handle(req: &super::shared::BenchmarkRequest) -> super::shared::BenchmarkResponse {
    let action = req.param("action");
    let config = load_config();
    let result = format!("Action {} with config loaded", action);
    super::shared::BenchmarkResponse::ok(&result)
}

fn load_config() -> String {
    "config_loaded".to_string()
}
